src/midgard/polyline2.cc

```diff
@@ -180,6 +16,7 @@ template void
 Polyline2<PointLL>::Generalize(std::list<PointLL>&, float, const std::unordered_set<size_t>&);
 template void
 Polyline2<Point2>::Generalize(std::list<Point2>&, float, const std::unordered_set<size_t>&);
+*/
```

Comment:
With these lines commented, there's nothing left in the file. Why not delete it?

Reply:
Ah, forgot to do this one - fixed.